circle.yml

```diff
@@ -46,14 +46,42 @@ jobs:
       DOCKER_USERNAME: travisciresin
```

Comment:
Would it makes sense to abstract the standard env stuff to a default env section of the yaml?